static/js/controllers/inbox.js

```diff
@@ -607,37 +615,49 @@ var feedback = require('../modules/feedback'),
         $rootScope.$broadcast.apply($rootScope, arguments);
       };
 
-      // TODO promisify DeleteDoc
-      var _deleteDoc = function(id) {
-        var deferred = $q.defer();
-        if (!id) {
-          return deferred.reject('Error deleting document : no docToDeleteId set');
+      $scope.deleteDoc = function(docs) {
+        if (!docs) {
+          return;
+        }
+        if (!_.isArray(docs)) {
```

Comment:
Actually looking at the code we use `docs` outside of DeleteDoc as well for working out what to show in the modal, eg: "delete 2 docs?". Without the isArray check here we'd have to jump through hoops in the other two places.